### ISSUE TITLE
[jazzy] C++ Change in Storage API

### ISSUE THREAD

**Issue by DLu** [MetroRobots/classic_bags#14]
```
In file included from /tmp/ws/src/classic_bags/test/test_basic.cpp:39:
/tmp/ws/src/classic_bags/include/classic_bags/bag.hpp: In constructor ‘classic_bags::MessageInstance::MessageInstance(const rosbag2_storage::SerializedBagMessage&, const std::string&)’:
/tmp/ws/src/classic_bags/include/classic_bags/bag.hpp:64:37: error: ‘const struct rosbag2_storage::SerializedBagMessage’ has no member named ‘time_stamp’
   64 |     time_ = rclcpp::Time(serialized.time_stamp);
      |                                     ^~~~~~~~~~
```

**Comment by DLu** [comment]
Because of [this PR](https://github.com/ros2/rosbag2/pull/1531)

**Comment by DLu** [comment]
Closed by #15
